api/lib/domain/services/token-service.js

```diff
@@ -15,12 +15,11 @@ function createTokenForCampaignResults(userId) {
   }, settings.authentication.secret, { expiresIn: settings.authentication.tokenForCampaignResultLifespan });
 }
 
-function createTokenForStudentReconciliation({ userAttributes }) {
-  const { attributeMapping } = settings.saml;
+function createTokenForStudentReconciliation(externalUser) {
```

Comment:
identity = identifier, mais pas autoriser
```suggestion
function createIDTokenForStudentReconciliation(externalUser) {
```

Reply:
Done